tests/sentry/issues/test_escalating.py

```diff
@@ -142,8 +138,8 @@ def test_query_different_orgs(self) -> None:
         org_b = Factories.create_organization()
         proj_b = Factories.create_project(org_b)
 
-        event1 = self._load_event_for_group(project_id=proj_a, minutes_ago=60)
-        event_proj_org_b_1 = self._load_event_for_group(project_id=proj_b, minutes_ago=60)
+        event1 = self._load_event_for_group(project_id=proj_a, hours_ago=2)
+        event_proj_org_b_1 = self._load_event_for_group(project_id=proj_b, hours_ago=2)
```

Comment:
why was `event1` and `event_proj_org_b_1` changed from 1 hr (60 mins) to 2 hrs ago?

Reply:
Good catch!